stdlib/logging/__init__.pyi

```diff
@@ -31,13 +31,27 @@ class Filterer(object):
     def removeFilter(self, filter: _FilterType) -> None: ...
     def filter(self, record: LogRecord) -> bool: ...
 
+class Manager(object):
+    root: RootLogger
+    disable: int
+    emittedNoHandlerWarning: bool
+    loggerDict: dict[str, Type[Logger]]
+    loggerClass: Optional[Type[Logger]]
+    logRecordFactory: Optional[Callable[..., LogRecord]]
+    def __init__(self, rootnode: RootLogger) -> None: ...
+    def getLogger(self, name: str) -> Type[Logger]: ...
```

Comment:
This returns a `Logger` instance, not a class. `PlaceHolder`s are never returned.

```suggestion
    def getLogger(self, name: str) -> Logger: ...
```